src/routes/users/components/UpdateUserForm.tsx

```diff
@@ -105,6 +116,10 @@ const UpdateUserForm = ({
         });
     }, [onCopy, toast]);
 
+    useEffect(() => {
+        return dispatchWithAutoAbort(listUsers({ page, ordering, match }));
```

Comment:
I think you don't need page, ordering or match here. You want to check last admin across all users after all.

Reply:
Can also potentially use dispatchWithAutoAbort on retrieveUser and remove normal dispatch

Reply:
Nice catch, I agree on the first point and made the changes. Regarding `retrieveUser`, we need to use a normal dispatch instead of `dispatchWithAutoAbort` to avoid an empty list when having the drawer open